tests/TestData/mutedProblems.json

```diff
@@ -1532,6 +1532,41 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Directory",
+      "methods": [
+        {
+          "name": "close",
+          "problems": [
+            "parameter mismatch"
+          ]
+        },
+        {
+          "name": "rewind",
+          "problems": [
+            "parameter mismatch"
+          ]
+        },
+        {
+          "name": "read",
+          "problems": [
+            "parameter mismatch"
+          ]
+        }
+      ]
+    },
+    {
```

Comment:
This should be removed and the stub should be changed to `private __construct()`.